Devices/UbermapDevices.py

```diff
@@ -20,8 +20,18 @@ def get_device_name(self, device):
         name = device.class_display_name
         if self.cfg.get('use_md5'):
             params = ''
-            for i in device.parameters[1:]:
-                params += i.original_name
+
+            '''
+	        Creates cfg without hash for Massive from params 10+
+	        only works if macros set to params 2-9 (default)
+			'''
+
+            if name == 'Massive':
+            	for i in device.parameters[10:]:
+                	params += i.original_name
```

Comment:
Indentation on this line doesn't match the rest of the file - perhaps it is using tabs rather than spaces? Indentation should be 4 spaces